config/manager/manager.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1beta1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: system
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+  labels:
+    control-plane: controller-manager
+spec:
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        control-plane: controller-manager
+    spec:
+      containers:
+        - command:
+            - /manager
+          args:
+            - --enable-leader-election
+          image: controller:latest
+          name: manager
+          resources:
+            limits:
+              cpu: 100m
```

Comment:
Set a higher value for the limits compared to requests.
As a start, we can set requests to CPU 50m, Mem 64Mi, and limits to CPU 200m, Mem 128Mi